test/Assert.t.sol

```diff
@@ -25,4 +26,39 @@ contract TestAsserts is Test, HelperAssert {
         string memory reason = "Testing assertion";
         t(true, reason);
     }
-}
+
+    function test_HelperAssert_t_true() public {
```

Comment:
the old tests (test_TFail and test_TSuccess) can be replaces with these new ones